examples/soupbintcp-client/src/main/java/com/paritytrading/nassau/soupbintcp/client/Client.java

```diff
@@ -57,9 +56,9 @@ public static Client connect(SocketAddress address) throws IOException {
 
     @Override
     public void message(ByteBuffer buffer) {
-        histogram.recordValue(System.nanoTime() - sentAtNanoTime);
+        histogram.recordValue(System.nanoTime() - buffer.asLongBuffer().get());
```

Comment:
Please use `ByteBuffer#getLong()` rather than `asLongBuffer()` to avoid unnecessary `LongBuffer` allocations.